### PR TITLE
Fix/functional programming challange

### LINKED ISSUE #38513: missing word in challange question
hello it’s not a big deal but I felt I should report it so the challenge becomes more clear
in Functional Programming: Pass Arguments to Avoid External Dependence in a Function challenge
should ask the user to return the value after increment, not just increment
challenge 
link: https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/functional-programming/pass-arguments-to-avoid-external-dependence-in-a-function

### PR DESCRIPTION
**Pull request by kundan59**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #38513

Added missing "return" word to make the challenge more clear.

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/pass-arguments-to-avoid-external-dependence-in-a-function.english.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Finally, the function would always produce the same output for the same set of i
 ## Instructions
 <section id='instructions'>
 Let's update the <code>incrementer</code> function to clearly declare its dependencies.
-Write the <code>incrementer</code> function so it takes an argument, and then increases the value by one.
+Write the <code>incrementer</code> function so it takes an argument, and then return a result after increases the value by one.
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/pass-arguments-to-avoid-external-dependence-in-a-function.english.md:21]:
```suggestion
Write the <code>incrementer</code> function so it takes an argument, and then returns a result after increasing the value by one.
```
Minor grammar nitpicks.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/pass-arguments-to-avoid-external-dependence-in-a-function.english.md (hunk 1 above)
Comment (kundan59) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/pass-arguments-to-avoid-external-dependence-in-a-function.english.md]: @ojeytonwilliams update Pr after eliminating grammatical mistakes. Please have a look thanks.

**Comment by kundan59** [comment]
@ojeytonwilliams update Pr after eliminating grammatical mistakes. Please have a look thanks.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM 👍

**Review by moT01 — APPROVED** [review]
LGTM @kundan59 🎉 Thank you for contributing to freeCodeCamp 🎉

**Pull request merged**